requirements.in

```diff
@@ -1,3 +1,5 @@
 gh_issues
 render_engine
 typer
+pip-tools
+gh-issues
```

Comment:
This is already present, but I believe it needs to be updated with `pip-compile -P gh-issues` or `pip-compile -U`.